testblas/level1double.go

```diff
@@ -407,6 +407,46 @@ var DoubleOneVectorCases = []DoubleOneVectorCase{
 		},
 	},
 	{
+		Name:   "EmptyStationary",
```

Comment:
Can we say "EmptyZero" or "EmptyIncZero"? On its own EmptyStationary is confusing.